app/gui2/src/stores/graph.ts

```diff
@@ -30,6 +32,8 @@ export const useGraphStore = defineStore('graph', () => {
   const nodes = reactive(new Map<ExprId, Node>())
   const exprNodes = reactive(new Map<ExprId, ExprId>())
 
+  const unconnectedEdge: Ref<UnconnectedEdge | null> = ref(null)
```

Comment:
prefer `ref<UnconnectedEdge>()` instead. the `undefined` will be added by the overload.
(same applies in other places with `: Ref<T | null> = ref(null)`)